### ISSUE TITLE
Publish fails to chmod when dll has space in filename

### ISSUE THREAD

**Issue by carbonrobot** [aws/aws-extensions-for-dotnet-cli#19]
I'm using version 2.1.4 of Amazon.Lambda.Tools. When an assembly has a space in the file name, the publish command will fail to chmod the assembly properly.

csproj
```
    <Reference Include="HAT Common">
      <HintPath>..\lib\HAT Common.DLL</HintPath>
    </Reference>
```

Output from publish
```
//
Changed permissions on published dll (chmod +r CommonDatastream.dll).
... publish: /bin/chmod: cannot access 'HAT': No such file or directory
Changed permissions on published dll (chmod +r Newtonsoft.Json.dll).
//
```

The chmod file name needs to be quoted when run in shell.

https://github.com/aws/aws-extensions-for-dotnet-cli/blob/dev/src/Amazon.Lambda.Tools/LambdaDotNetCLIWrapper.cs#L174

**Comment by normj** [comment]
Thanks for letting us know. I'll make that change to quote it.

**Comment by normj** [comment]
This was fixed in the 3.0.1 Global tools version of Amazon.Lambda.Tools. For more info on the Global Tools switch read [here](https://aws.amazon.com/blogs/developer/net-core-global-tools-for-aws/)
